docs/api.md

```diff
@@ -1067,27 +1153,249 @@ Example Response Body:
 
 ```json
 {
-  "owner": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc3",
+  "owner": "0x77a035b677d5a0900e4848ae885103cd49af9633",
   "balances": {
     "PBTC-USDC": {
-      "margin": "120000",
-      "position": "20",
-      "indexValue": "6000000000000000000000",
-      "indexTimestamp": "1585933964",
-      "cachedMargin": "12005",
+      "cachedMargin": "40181034",
+      "margin": "40169512",
+      "position": "10000",
+      "pendingMargin": "0",
+      "pendingPosition": "0",
+      "indexValue": "0.057858741951992068",
+      "indexTimestamp": "1588271672"
     }
   }
 }
 ```
 
-### Account Response Object
+#### Account Response Object
+
+| Field Name      | Description                                                                    |
+|-----------------|--------------------------------------------------------------------------------|
+| owner           | The user's wallet address.                                                     |
+| balances        | A mapping of the market to the balances for that market                        |
+| margin          | This is calculated as `cachedMargin- (indexValue - globalIndexValue)*position` |
+| position        | The balance in position token (e.g. PBTC).                                     |
+| cachedMargin    | This is the last stored margin value                                           |
+| pendingMargin   | This is the (pending) component of the margin when a fill is still pending     |
+| pendingPosition | This is the (pending) component of the position when a fill is still pending   |
+| indexValue      | The value of the global index from the last interaction with the account       |
+| indexTimestamp  | The timestamp when the index value was set                                     |
+
+
+### GET `/v1/perpetual-accounts`
+
+Description:
+This endpoint returns balances for all perpetual accounts.
+
+Query Params:
+| Field Name     | Description                                                     |
+|----------------|-----------------------------------------------------------------|
+| isLiquidatable | If set to true, returns accounts that are yet to be liquidated  |
```

Comment:
nit `If set to true, returns accounts that are below the margin requirement.`

Reply:
added